setup.cfg

```diff
@@ -5,6 +5,26 @@ author_email = info@ansible.com
 summary = "A tool for building Ansible Execution Environments"
 home_page = https://ansible-builder.readthedocs.io
 description_file = README.md
+description_content_type = text/markdown
+license_file = LICENSE.md
+license = Apache Software License, Version 2.0
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: System Administrators
+    License :: OSI Approved :: Apache Software License
+    Natural Language :: English
+    Operating System :: POSIX
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3 :: Only
+    Topic :: Software Development :: Libraries :: Python Modules
```

Comment:
This one doesn't seem right. One does not `import ansible_builder` code, which is how I read this classifier.